apps/nextjs/src/components/DialogControl/ContentOptions/AdditionalMaterialsUserFeedback.tsx

```diff
@@ -0,0 +1,37 @@
+import { OakBox, OakFlex } from "@oaknational/oak-components";
+
+import FeedBack from "@/components/Feedback";
+import { usePosthogFeedbackSurvey } from "@/hooks/surveys/usePosthogFeedbackSurvey";
+
+const AdditionalMaterialsUserFeedback = ({
+  closeDialog,
+}: {
+  readonly closeDialog: () => void;
+}) => {
+  const { survey, submitSurvey, closeDialogWithPostHogDismiss } =
+    usePosthogFeedbackSurvey({
+      closeDialog,
+      surveyName: "Teaching material feedback",
```

Comment:
Be really great to check these exist at build time -- and have them typed. Especially the more we create.